src/utils/youtubeUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { google, youtube_v3 } from 'googleapis';
+import { logger } from './utils';
+
+// initialize the Youtube API library
+const youtube = google.youtube({
+    version: 'v3',
+    auth: 'AIzaSyA97mUqVeEm2Zxfntt7A6SZAmP_bkYKPKQ',
+});
+
+export async function searchYoutube(request: string): Promise<youtube_v3.Schema$SearchListResponse> {
+    return await youtube.search.list({
+        part: 'id,snippet',
+        q: request,
+    })
+    .then((res) => {
```

Comment:
do you need to use promise-like then after the await? I'm thinking of something like
`res = await youtube.search.list
.catch(...)`

Reply:
I did this , thanks for reviewing !